### DISCUSSION TITLE
Use Formula Field as table display column

### DISCUSSION THREAD

**Discussion by tobehn** [Budibase/budibase#2641] [Help] [answered]
I created an extra filed `sign` in a bb-table, hyphenating to fields `number` and `name` as a Formula Field. I want this field be set _use as table display column_ (with which it automatically will be marked as required). 
When i then try to create a new row it says "_sign cannot be empty_" due to the requirement setting i guess.
Whereas i do have both other fields set and the hyphenation could possibly be done on creation.

If this is not the right way - is there another possibilty to achive this?

**Comment by mjashanks** [comment, marked as answer]
Hey @tobehn - I can reproduce this. This is a bug.

Either
1. It should work, as you describe
OR
2. Forumula fields should not be allowed as display columns

Hopefully, No.1 is the correct answer! I will log this.

**Reply by tobehn** [reply]
@mjashanks Thank you for the quick response. I saw you already created a bug report. 👍
